Gruntfile.js

```diff
@@ -404,6 +394,64 @@ module.exports = (grunt) => {
 
   let child = null
 
+  grunt.registerTask('copy:frontend', function () {
+    grunt.task.run(['copy:htmlFiles', 'copy:assets', 'copy:strings'])
+  })
+
+  grunt.registerTask('copy:contracts', async function () {
+    const done = this.async()
+    const { contractsVersion } = packageJSON
+
+    // NOTE: the latest version
+    await mkdir(`${distContracts}/${contractsVersion}`)
+    for (const dirent of await readdir(distContracts, { withFileTypes: true })) {
+      if (dirent.isFile()) {
+        const fileName = dirent.name
+        await copyFile(`${distContracts}/${fileName}`, `${distContracts}/${contractsVersion}/${fileName}`)
+        await rm(`${distContracts}/${fileName}`)
+      }
+    }
+
+    // NOTE: all previously pinned versions
+    const versions = (await readdir('contracts', { withFileTypes: true })).filter(dirent => {
+      return dirent.isDirectory() && dirent.name !== contractsVersion
+    }).map(dirent => dirent.name)
+    for (const version of versions) {
+      await cp(`contracts/${version}`, `${distContracts}/${version}`, { recursive: true })
+    }
+
+    done()
+  })
+
+  grunt.registerTask('chelDeploy', function () {
+    const tasks = !production ? ['exec:chelDevDeploy'] : ['exec:chelProdDeploy']
+    grunt.task.run(tasks)
+  })
+
+  grunt.registerTask('run', function () {
+    const done = this.async() // Tell Grunt we're async.
+
+    grunt.log.writeln('backend: running...')
+    const child = fork(backendIndex, process.argv, {
+      env: { NODE_ENV, ...process.env, DB_ROOT_PATH: `/${distDir}` },
+      execArgv: ['--require', '@babel/register']
+    })
+    child.on('error', (err) => {
+      if (err) {
+        console.error('error starting or sending message to child:', err)
+        process.exit(1)
+      }
+    })
+    child.on('exit', (c) => {
+      if (c !== 0) {
+        grunt.log.error(`child exited with error code: ${c}`.bold)
+        // ^C can cause c to be null, which is an OK error.
+        process.exit(c || 0)
+      }
+    })
+    done()
+  })
```

Comment:
The `'backend:launch'` task is more lightweight, as it avoids spinning off a new nodejs process...

Is there a reason this `run` task was created instead of reusing the existing `'backend:launch'` task?

Reply:
Actually, I tried to use `backend:launch`, and got an error saying `can not find ./backend/index.js`. Now, I've fixed that error and I am going to use `backend:launch`.